### PR TITLE
RF-1.0.1.10.1 - Installing envs from user repos

### PR DESCRIPTION
**Pull request by asa1997**
### To raise a new Pull Request, the following prerequisites need to be met. Please tick before proceeding:

- [ ] a conversation was held in the appropriate media [hyperledgerkochi](https://github.com/orgs/hyperledgerkochi/projects/3) channel.
- [x] a Github [Issue](https://github.com/hyperledgerkochi/KOBman/issues) was opened for this feature / bug.
- [ ] test coverage was added .

**This addresses Issue** #162

### CODE REVIEW

**Comment by asa1997** [comment]
@EtricKombat Please review this pull request and test for errors. Make a release with the changes in this pull for testing.

Instructions to test
--------------------
1. Open your user-config.cfg file unser .kobman/etc
2. Add the following user configs into the file, `KOBMAN_ENV_REPOS=<user_namespace>/user_env_repo>`(You may use my test repo `asa1997/test_repo_a` https://github.com/asa1997/test_repo_a. You can also create one like in the link)
3. Run source command. `source .kobman/bin/kobman-init.sh'
4. Run the update command. `kob update`

Output
---------
- If you are testing it from my test repository. the following o/p will be generated.

Screenshot from 2020-08-25 13-46-13

- Make the env variable `KOBMAN_ENV_REPOS` empty under .kobman/etc/user-config.cfg
   - Run the source command `source .kobman/bin/kobman-init.sh`
   - Run update `kob update`
Screenshot from 2020-08-25 13-49-14

**Comment by EtricKombat-zz** [comment]
@asa1997  this has been tested out . Please find the snapshots 

image

After the clean up of .kobman/etc/user-config.cfg

image

File: src/main/bash/kobman-env-helpers.sh [hunk 1]
```diff
@@ -97,6 +97,66 @@ function __kobman_create_fork
     return 0
   fi
 }
+function __kobman_download_envs_from_repo
+{
+  # __kobman_echo_white "Downloading environments from external repos"
```
Comment (EtricKombat-zz) [inline src/main/bash/kobman-env-helpers.sh:102]: @asa1997  , if not required . can be removed along with next pull request

**Review by EtricKombat-zz — COMMENTED** [review]
@asa1997  , needs to remove all the unwanted commented line along with the next pull request.

**Comment by EtricKombat-zz** [comment]
@panickervinod , @sumodgeorge  , this  pull request has been reviewed.

LGTM

**Comment by panickervinod** [comment]
@asa1997  Merging this one based on LGTM & manual testing. Do create an issue to include test case for the commands functionality

**Pull request merged**